data/campaigns/Heir_To_The_Throne/scenarios/01_The_Elves_Besieged.cfg

```diff
@@ -581,6 +581,27 @@
             name=besieged_enemies_killed
             add=1
         [/set_variable]
+
+        # httt achievement one
+        {IF_VAR besieged_enemies_killed equals 2 (
+            [then]
+                [set_achievement]
+                    content_for=Heir_To_The_Throne_achievements
+                    id="HttT_S1_Defend_Aethenwood"
+                    sound="fanfare-short.wav"
+                [/set_achievement]
```

Comment:
I wonder if it might be possible/worthwhile to come up with a macro here to wrap these tags to reduce some of the repetition? I mean, since it looks like it's the same sound effect every time and all...